docs/server/upgrading.rst

```diff
@@ -179,6 +179,15 @@ To perform the upgrade follow the next steps:
 Post-upgrade adjustments
 ------------------------
 
+* Check that you can login as an admin user.  If you existing adin user has no
+  email address then you will not be able to login.  In order the fix this use
+  :djadmin:`createsuperuser` to create a new superusers, or to create a
+  temporary superuser to fix your existing one.
+
+  .. code-block:: bash
+
+     (env) $ pootle createsuperuser
```

Comment:
I didn't want to fiddle those docs anymore.  I have some upgrade cleanups and will deal with that with those.